nautobot/utilities/forms/fields.py

```diff
@@ -454,6 +454,8 @@ def get_choices():
         # Choices are fetched each time the form is initialized
         super().__init__(label="Tags", choices=get_choices, required=False, *args, **kwargs)
 
+    to_field_name = "slug"
```

Comment:
What'll we need to do in 2.0 to avoid reintroducing the bug, given that tags don't have slugs any more?

Reply:
IIRC `name` since that's the unique field in 2.0. This keeps things unbreaking.